.github/workflows/create-release.yml

```
@@ -216,3 +216,15 @@ jobs:
                     install_sct-${{ github.event.inputs.milestone_title }}_macos.sh,\
                     sct_apptainer_${{ github.event.inputs.milestone_title }}.tar.gz"
         draft: true
+
+    - name: Update `stable` docs branch to point to new release
+      # - We use a manual `stable` branch instead of ReadTheDocs' automated `stable` system.
+      # - This allows us to backport simple doc changes (e.g. typo fixes) to our `stable` branch
+      #   without having to create any new tags or releases.
+      # - When a new release is created, it becomes the new base of the `stable` branch.
+      # - NOTE: This will wipe out all the previous backported changes, so make sure to also update `master`
+      #   whenever we backport changes (so that they will be preserved when resetting the branch to the release).
+      run: |
+        git checkout stable
+        git reset --hard ${{ github.event.inputs.milestone_title }}
+        git push --force
```

Note that it's possible to change the commit that a (non-checked-out) branch points to directly, with `git branch -f`:
```
        git branch --force stable ${{ github.event.inputs.milestone_title }}
        git push --force stable
```

But also, `git push` allows pushing an arbitrary local commit directly to a remote branch name, with the `local:remote` syntax. Combined with specifying `origin` and `refs/tags/...` and `refs/heads/...` for extra safety, this gives:
```suggestion
        git push --force origin refs/tags/"${{ github.event.inputs.milestone_title }}":refs/heads/stable
```

---

> Note that it's possible to change the commit that a (non-checked-out) branch points to directly, with `git branch -f`:

Thanks for the heads up about [`git branch --force`](https://git-scm.com/docs/git-branch#Documentation/git-branch.txt---force):

> > Reset `<branch-name>` to `<start-point>`, even if `<branch-name>` exists already. Without `-f`, `git branch` refuses to change an existing branch.